pylint/checkers/typecheck.py

```diff
@@ -335,6 +335,11 @@ def _emit_no_member(node, owner, owner_name, ignored_mixins):
             return False
         if not all(map(has_known_bases, owner.type.mro())):
             return False
+    if node.attrname.startswith('_' + owner_name):
+        # Test if an attribute has been mangled ('private' attribute)
+        unmangled_name = node.attrname.split('_' + owner_name)[-1]
+        if owner.getattr(unmangled_name, None) is not None:
```

Comment:
Please pass the second argument as a keyword argument. Whenever we pass numbers, strings, None, it is more obvious what they mean if we attach an additional context to them through the keyword argument

Reply:
Also make sure this doesn't raise `astroid.AstroidNotFoundError`!